addons/partner_autocomplete/models/res_company.py

```diff
@@ -39,10 +32,7 @@ def _get_view(self, view_id=None, view_type='form', **options):
         arch, view = super()._get_view(view_id, view_type, **options)
 
         if view_type == 'form':
-            for i, node in enumerate(arch.xpath("//field[@name='name' or @name='vat']")):
-                if i == 0:
-                    node.addnext(E.field(name='partner_gid', invisible='1'))
-
+            for i, node in enumerate(arch.xpath("//field[@name='name' or @name='vat' or @name='duns']")):
```

Comment:
This field was removed. It looks like I missed that in the `master` PR, but it shouldn't matter :shrug:.
```suggestion
            for i, node in enumerate(arch.xpath("//field[@name='name' or @name='vat']")):
```